addons/pos_restaurant/static/src/js/ChromeWidgets/TicketButton.js

```diff
@@ -17,12 +17,22 @@ odoo.define('pos_restaurant.TicketButton', function (require) {
             }
             async _syncAllFromServer() {
                 const pos = this.env.pos;
+                const pos_config_id = pos.config.id
                 try {
-                    for (const floor of pos.floors) {
-                        for (const table of floor.tables) {
-                            await pos.replace_table_orders_from_server(table);
-                        }
-                    }
+                    const server_orders = await this.rpc({
+                        model: 'pos.order',
+                        method: 'get_all_table_draft_orders',
+                        args: [[], pos_config_id],
+                        kwargs: {context: pos.session.user_context},
+                    }, {
+                        timeout: 7500,
+                        shadow: false,
+                    });
+                    const orders = Object.keys(pos.tables_by_id).reduce(function (acm, table_id){
+                        const orders = pos.get_table_orders(pos.tables_by_id[table_id]);
+                        return acm.concat(orders);
+                    },[])
```

Comment:
I checked how we get every order in the ticket screen and this is it:
```suggestion
                    const orders = this.env.pos.get('orders').models
```

Reply:
@trj-odoo 

If I do this change, as here:

https://github.com/odoo/odoo/pull/106371

A duplicated new order is created each time `Orders` is clicked:

https://youtu.be/6n3oOkUZ_PM

cc @hugho-ad

Reply:
Yes indeed, my bad. When replacing the orders, we are destroying them which directly affect the `pos.get('orders').models` during the forEach in the `_replace_orders` call (mutating the array during a loop is bad)